src/components/Pages/ManageWiki/Features/ItemsTable.vue

```diff
@@ -0,0 +1,216 @@
+<template>
+  <v-data-table
+	:headers="headers"
+	:items="wikibaseItems"
+	sort-by="wikidata"
+	class="elevation-1"
+  >
+	<template v-slot:top>
+	  <v-toolbar
+		flat
+	  >
+		<v-toolbar-title>Items</v-toolbar-title>
+		<v-divider
+		  class="mx-4"
+		  inset
+		  vertical
+		></v-divider>
+		<v-spacer></v-spacer>
+		<v-dialog
+		  v-model="dialog"
+		  max-width="500px"
+		>
+		  <template v-slot:activator="{ on, attrs }">
+			<v-btn
+			  color="primary"
+			  dark
+			  class="mb-2"
+			  v-bind="attrs"
+			  v-on="on"
+			>
+			  Add New Mapping
```

Comment:
Not sure if this really matters but in the mockups the buttons are below the tables :man_shrugging:

Reply:
Moved it